### PR TITLE
Generate swagger from update-codegen

### PR DESCRIPTION
**Pull request by thockin**
Swagger "docs" are actually Go code, which is used by other codegen tools, so if you really want to regen EVERYTHING, this is part of it and sequence matters.

This includes #115245 just because of the diff context.

/kind cleanup
/sig architecture

```release-note
NONE
```

### CODE REVIEW

File: hack/make-rules/update.sh [hunk 1]
```diff
@@ -36,7 +36,6 @@ if ! ${ALL} ; then
 fi
 
 BASH_TARGETS=(
-	update-generated-protobuf
```
Comment (aojea) [inline hack/make-rules/update.sh:39]: commented on https://github.com/kubernetes/kubernetes/pull/115245

File: hack/verify-generated-swagger-docs.sh [hunk 2]
```diff
@@ -1,46 +0,0 @@
...
-kube::util::trap_add "git worktree remove -f ${_tmpdir}" EXIT
-cd "${_tmpdir}"
-
-# Update the generated swagger docs
-hack/update-generated-swagger-docs.sh
-
-# Test for diffs
-diffs=$(git status --porcelain | wc -l)
-if [[ ${diffs} -gt 0 ]]; then
-  echo "Generated swagger type documentation is out of date:" >&2
```
Comment (aojea) [inline hack/verify-generated-swagger-docs.sh:42]: how is this verified now?

File: hack/make-rules/update.sh (hunk 1 above)
Comment (thockin) [reply hack/make-rules/update.sh:39]: resolved there, I think?

File: hack/verify-generated-swagger-docs.sh (hunk 2 above)
Comment (thockin) [reply hack/verify-generated-swagger-docs.sh:42]: worktree, run, diff

Comment (aojea) [reply hack/verify-generated-swagger-docs.sh:42]: who runs that and where?

Comment (thockin) [reply hack/verify-generated-swagger-docs.sh:42]: verify-codegen.sh is called from `make verify`

Comment (aojea) [reply hack/verify-generated-swagger-docs.sh:42]: and it runs 
```
export UPDATE_API_KNOWN_VIOLATIONS=true
"${KUBE_ROOT}/hack/update-codegen.sh" --verify-only "$@"
```

Comment (aojea) [reply hack/verify-generated-swagger-docs.sh:42]: I was not using latest bits on my local branch, that is why I didn't find the verify code

Comment (thockin) [reply hack/verify-generated-swagger-docs.sh:42]: yeah, man.  rebase your local tree :)

File: staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go [hunk 3]
```diff
@@ -397,9 +395,9 @@ func importOrder(deps map[string][]string) ([]string, error) {
 	if len(remainingNodes) > 0 {
 		return nil, fmt.Errorf("cycle: remaining nodes: %#v, remaining edges: %#v", remainingNodes, graph)
 	}
-	for _, n := range sorted {
-		fmt.Println("topological order", n)
-	}
+	//for _, n := range sorted {
```
Comment (dims) [inline staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go:398]: can we delete this commented out code?

**Pull request merged**